docs/api-reference/next/image.md

```diff
@@ -86,18 +56,18 @@ Required, except for statically imported images, or those with [`layout="fill"`]
 
 ## Optional Props
 
-The `<Image />` component optionally accepts the following properties.
+The `<Image />` component accepts a number of additional properties beyond those which are required. This section describes the most commonly-used properties of the Image Component.
```

Comment:
Most common – or all of them? Given it's the API reference, I would expect all options 👍

Reply:
Yeah this is a little confusing because this section of the doc covers the most-commonly used props, while all props are covered in the doc as a whole. I'll come up with a way to clarify

Reply:
Added a link to the "advanced props" section to make that a little more explicit